cmd/mobile/main.go

```diff
@@ -108,10 +109,13 @@ func main() {
 	}
 }
 
+// NewClientsOrDie creates a new set of clients for Kubernetes, Service Catalog and Mobile Services
+// if any of these clients fails to create then the process wil die.
 func NewClientsOrDie(configLoc string) (kubernetes.Interface, m.Interface, sc.Interface) {
 	config, err := clientcmd.BuildConfigFromFlags("", configLoc)
 	if err != nil {
-		panic(err.Error())
+		fmt.Fprintln(os.Stderr, err.Error())
```

Comment:
how clean is the error produced. Do we want to add some context to it

Reply:
I think it looks quite clean:
```sh
 $ ./mobile
stat /Users/pbrookes/.kube/config: no such file or directory
```